content/community/action.md

```diff
@@ -50,8 +132,10 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Bloomberg" image="/images/logos/bloomberg.png" >}}{{< /company >}}
```

Comment:
already listed with quote

Reply:
All Companies with Quotes have been listed with the Logos, so we can make presentations and keep them together

Reply:
great! let's click on "resolve" near my comments and check that all companies doubled then (I saw the companies only listed once with quote :))